src/solve.ts

```diff
@@ -213,7 +213,7 @@ function reconize(dir: string): Promise<string> {
                 if (end_of_speech) {
                     const result = rec
                         .result()
-                        .alternatives.sort((a, b) => b.confidence - a.confidence)[0].text;
+                        .alternatives.sort((a:any, b:any) => b.confidence - a.confidence)[0].text;
```

Comment:
```suggestion
                        .alternatives.sort((a, b) => b.confidence - a.confidence)[0].text;
```

`a` and `b` type is `RecognitionResults | (RecognitionResults & SpeakerResults)`.
I think not need this `any` cast.